### ISSUE TITLE
Look controls magicWindowTrackingEnabled:false disables up and down look on touch screen.

### ISSUE THREAD

**Issue by dotnetrix** [aframevr/aframe#4775]
When setting the look-controls magicWindowTrackingEnabled attribute to false the gyro based rotations stop as desired. 
However, on mobiles with gyroscopes we can no longer look up and down using touch drag.
On Desktop with mouse we are free to look in any direction we like.

- A-Frame Version: 1.1.0
- Platform / Device: android device with gyroscope
- Reproducible Code Snippet or URL: [(https://marapainter.com/test.htm)]

**Comment by dmarcos** [comment]
This is a design decision. Look controls don't implement vertical panning. You can implement it by deriving your own controls from the [look-controls component code](https://github.com/aframevr/aframe/blob/master/src/components/look-controls.js)

**Comment by dotnetrix** [comment]
I don't understand it being a design decision as it gives different behaviour on desktop and mobile. I had a peek at the look controls code and it's far too complex for my needs. I have already implemented my own movement controls (I need to use buttons to move around on the ground in the direction of view) so I'll just add a look flag and implement that too. It's just that Look worked so well on desktop that I didn't think I needed to.
